Xamarin.Forms.Core/ListView.cs

```diff
@@ -46,6 +46,10 @@ public class ListView : ItemsView<Cell>, IListViewController, IElementConfigurat
 
 		public static readonly BindableProperty SeparatorColorProperty = BindableProperty.Create("SeparatorColor", typeof(Color), typeof(ListView), Color.Default);
 
+		public static readonly BindableProperty HorizontalScrollBarVisibilityProperty = BindableProperty.Create(nameof(HorizontalScrollBarVisibility), typeof(ScrollBarVisibility), typeof(ScrollView), ScrollBarVisibility.Default);
```

Comment:
I think the target type here should be `typeof(ListView)` rather than `typeof(ScrollView)`.